html/modules/custom/rwr_sitrep/rwr_sitrep.module

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Helpers for the sitrep module.
+ */
+
+use Drupal\Core\Cache\RefinableCacheableDependencyInterface;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_menu_local_tasks_alter().
+ *
+ * Remove or adjust 'Reports' local task for operations or clusters.
+ * If this is an operation, have the link lead to that operation's situation
+ * reports only.
+ * If this is a cluster, remove the link unless the cluster has the Situation
+ * Report tag, in which case change the label accordingly.
+ *
+ * @see rwr_sitrep.links.task.yml
+ */
+function rwr_sitrep_menu_local_tasks_alter(&$data, $route_name, RefinableCacheableDependencyInterface &$cacheability) {
+
+  if ($group = \Drupal::routeMatch()->getParameter('group')) {
+    $sitrep_tid = \Drupal::state()->get('rwr_sitrep:sitrep_tid');
```

Comment:
This logic can be removed if the label for the tab is part of the term

Will become something like `$group->field_group_type->entity->get('field_tab_label')->value and translations will be handled on the term.